docs/content/examples/tutorials/basic-installation.md

```diff
@@ -84,142 +79,3 @@ services:
-    ```
-
-    Then don't forget to change the serial number and to restart the service.
-
-5. [Generate DKIM keys][docs-dkim] for your domain via `./setup.sh config dkim`.
-
-    Copy the content of the file `docker-data/dms/config/opendkim/keys/example.com/mail.txt` and add it to your DNS records as a TXT like SPF was handled above.
-
-    I use [bind9](https://github.com/docker-scripts/bind9) for managing my domains, so I just paste it on `example.com.db`:
-
-    ```txt
-    mail._domainkey IN      TXT     ( "v=DKIM1; h=sha256; k=rsa; "
-            "p=MIIBIjANBgkqhkiG9w0BAQEFACAQ8AMIIBCgKCAQEAaH5KuPYPSF3Ppkt466BDMAFGOA4mgqn4oPjZ5BbFlYA9l5jU3bgzRj3l6/Q1n5a9lQs5fNZ7A/HtY0aMvs3nGE4oi+LTejt1jblMhV/OfJyRCunQBIGp0s8G9kIUBzyKJpDayk2+KJSJt/lxL9Iiy0DE5hIv62ZPP6AaTdHBAsJosLFeAzuLFHQ6USyQRojefqFQtgYqWQ2JiZQ3"
-            "iqq3bD/BVlwKRp5gH6TEYEmx8EBJUuDxrJhkWRUk2VDl1fqhVBy8A9O7Ah+85nMrlOHIFsTaYo9o6+cDJ6t1i6G1gu+bZD0d3/3bqGLPBQV9LyEL1Rona5V7TJBGg099NQkTz1IwIDAQAB" )  ; ----- DKIM key mail for example.com
-    ```
-
-6. Get an SSL certificate, [we have a guide for you here][docs-ssl] (_Let's Encrypt_ is a popular service to get free SSL certificates).
-
-7. Start `docker-mailserver` and check the terminal output for any errors: `docker-compose up`.
-
-8. Create email accounts and aliases:
-
-    !!! example "With `SPOOF_PROTECTION=0`"
-
-        ```sh
-        ./setup.sh email add admin@example.com passwd123
-        ./setup.sh email add info@example.com passwd123
-        ./setup.sh alias add admin@example.com external-account@gmail.com
-        ./setup.sh alias add info@example.com external-account@gmail.com
-        ./setup.sh email list
-        ./setup.sh alias list
-        ```
-
-        Aliases make sure that any email that comes to these accounts is forwarded to your third-party email address (`external-account@gmail.com`), where they are retrieved (_eg: via third-party web or mobile app_), instead of connecting directly to `docker-mailserer` with POP3 / IMAP.
-
-    !!! example "With `SPOOF_PROTECTION=1`"
-
-        ```sh
-        ./setup.sh email add admin.gmail@example.com passwd123
-        ./setup.sh email add info.gmail@example.com passwd123
-        ./setup.sh alias add admin@example.com admin.gmail@example.com
-        ./setup.sh alias add info@example.com info.gmail@example.com
-        ./setup.sh alias add admin.gmail@example.com external-account@gmail.com
-        ./setup.sh alias add info.gmail@example.com external-account@gmail.com
-        ./setup.sh email list
-        ./setup.sh alias list
-        ```
-
-        This extra step is required to avoid the `553 5.7.1 Sender address rejected: not owned by user` error (_the accounts used for submitting mail to Gmail are `admin.gmail@example.com` and `info.gmail@example.com`_)
-
-9. Send some test emails to these addresses and make other tests. Once everything is working well, stop the container with `ctrl+c` and start it again as a daemon: `docker-compose up -d`.
```

Comment:
Another line that might be useful in `usage.md`.

I know there is a "Testing" section at the bottom with a few links. This should probably come earlier after completing the basic setup, and the actual details about testing with the linked tools could be on another page (Debugging maybe).